qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -196,7 +195,10 @@ def __init__(self,
             [1]: Boyer et al., Tight bounds on quantum searching
                  `<https://arxiv.org/abs/quant-ph/9605034>`_
         """
-        super().__init__(quantum_instance)
+        self._quantum_instance = None
```

Comment:
I am not sure why the base class needed to be eliminated given that this quantum instance/backend code has now been duplicated across the algos.